include/pcmki/pcmki_scheduler.h

```diff
@@ -58,7 +58,6 @@ struct rsc_ticket_s {
     int role_lh;
 };
 
-extern gboolean stage0(pe_working_set_t * data_set);
```

Comment:
stage0 is also mentioned in doc/sphinx/Pacemaker_Development/components.rst.

Reply:
Yeah I should have done that with this PR, but I saved all the doc changes for a later one